server/trucks.js

```diff
@@ -253,4 +254,14 @@ var foodTrucks = [
  * that method should take a single parameter that represents the day to filter on
  * use underscore's filter() method to return all trucks that have the day in their
  * schedule
- * /
\ No newline at end of file
+ */
+ 
+var filterByDay = function filterByDay(day) {
+
+	var todayFoodTrucks = _.filter(foodTrucks, function(truck){
+		for (var i = 0; i < truck.schedule.length; i++){
+			return day === truck.schedule[i];
```

Comment:
When using just a for loop, you're not going to want to call return in the loop until the condition is true: `if (return day === truck.schedule[i]) { return true; } By calling return without checking, it will return on the first iteration of the loop, and, unless the first day happens to be the day it is when executing the code, will return false and never return any trucks from filterByDay().